ports/servoshell/egl/ohos.rs

```diff
@@ -447,6 +448,24 @@ pub fn register_url_callback(cb: JsFunction) -> napi_ohos::Result<()> {
     Ok(())
 }
 
+#[napi]
+pub fn register_prompt_toast_callback(cb: JsFunction) -> napi_ohos::Result<()> {
+    debug!("register_prompt_toast_callback called!");
+    let tsfn: ThreadsafeFunction<String, ErrorStrategy::Fatal> =
+        cb.create_threadsafe_function(4, |ctx| {
```

Comment:
What does `4` mean here? Is it possible to turn this into a constant at the top of the file?

Reply:
The queue depth when calling the JS function (on the JS thread). I chose a constant per JsFunction instead, since suitable values will likely differ based on the specific function.